subsys/bluetooth/shell/bt.c

```diff
@@ -2897,7 +2897,7 @@ static int cmd_read_local_tx_power(const struct shell *sh, size_t argc, char *ar
 		}
 		err = bt_conn_le_get_tx_power_level(default_conn, &tx_power_level);
 		if (err) {
-			shell_print(sh, "Commad returned error error %d", err);
```

Comment:
"error error"

Reply:
@ndrs-pst are you planning to fix this in a separate PR, or is there still going to be an update to this one?

Reply:
I'll approve anyway, since this can be fixed incrementally too.